lib/wallet/http.js

```diff
@@ -1203,7 +1203,7 @@ class HTTP extends Server {
       res.setHeader('From-Cache', fromCache);
 
       if (broadcast) {
-        const tx = await req.wallet.sendMTX(mtx, passphrase);
+        const tx = await req.wallet.sendMTX(mtx, passphrase, true);
```

Comment:
nit: for boolean flags like this I would prefer to do:

```javascript
async sendMTX(tx) {
  return this._sendMTX(tx, false);
}
async sendMTXBlocking(tx) {
  return this._sendMTX(tx, true);
}
async _sendMTX(tx, sendBlocking);
```

Reply:
fixed